src/main/flight/rpm_filter.c

```diff
@@ -53,6 +53,7 @@ typedef struct rpmFilter_s {
     float minHz;
     float maxHz;
     float fadeRangeHz;
+    float fadeRangeHzInv;
```

Comment:
This is minor microoptimization. Reading additional value from memory will take same time, so improvement will be smaller than theoretical 13 cycle improvement. It will also increase register pressure, possibly further decreasing performance.
Also, inverse of frequency is time, cso maybe  fadeRangePeriod?

Reply:
fadeRangeHz is a difference between two frequencies. I'm not sure "period" is a good descriptor here.

Reply:
Thanks, I missed that.
Anyway, this part of PR is not worth complication of code.

Reply:
This consumes additional 4 bytes of RAM, possibly increases code size, adds 2 lines to code length and possibly causes problems when fadeRangeHz  and fadeRangeHzInv are not in sync.
Doing such unnecessary micro optimization will only make code harder to read and refactor.

 If you want to save some cycles  in notch filtering, implement vectorized (for vector3_t) version of biquad filter.